src/Lecture4.hs

```diff
@@ -232,9 +283,25 @@ our final result.
 If there's no some value (for example, there were not "Buy" products),
 you can return string "no value"
 -}
+showMaybeMax :: (Show a) => Maybe (Max a) -> String
+showMaybeMax Nothing = "no value"
+showMaybeMax (Just (Max x)) = show x
+
+showMaybeMin :: (Show a) => Maybe (Min a) -> String
+showMaybeMin Nothing = "no value"
+showMaybeMin (Just (Min x))= show x
 
 displayStats :: Stats -> String
-displayStats = error "TODO"
+displayStats (Stats sTotalPos sTotalSum sAbsMax sAbsMin sSellMax sSellMin sBuyMax sBuyMin sLong) = "\
+\Total positions:       : "++ (show . getSum $ sTotalPos) ++"\n\
```

Comment:
In this particular case, you don't need to use composition and dollars and clean up the code a little bit 🧹 
```suggestion
\Total positions:       : "++ show (getSum sTotalPos) ++"\n\
```